apps/synapse-portal-framework/src/components/standards/StandardsHeader.tsx

```diff
@@ -0,0 +1,121 @@
+import { Box, Typography, Button } from '@mui/material'
+import HeaderSearchBox from '../HeaderSearchBox'
+import Header from '../Header'
+import {
+  FeatureFlagEnum,
+  Query,
+  TextMatchesQueryFilter,
+} from '@sage-bionetworks/synapse-types'
+import { useGetFeatureFlag } from 'synapse-react-client/synapse-queries'
+
+const StandardsHeader = () => {
+  const searchPlaceholder = 'Search for a biomedical data standard'
+  const searchExampleTerms = [
+    'MRI data processing',
+    'Waveform interoperability',
+    'Ophthalmic imaging',
+    'Integration',
+    'Acquisition',
+    'File formats',
+    'Ontologies',
+    'Lab automation',
+  ]
+
+  const isFeatureEnabled = useGetFeatureFlag(
+    FeatureFlagEnum.PORTAL_SEARCH_HEADER,
+  )
```

Comment:
Is this feature flag check necessary for this portal?

Reply:
Probably not!